ajax/libs/swipebox/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "swipebox",
+  "version": "1.4.6",
+  "description": "A touchable jQuery lightbox",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/brutaldesign/swipebox.git"
+  },
+  "filename": "jquery.swipebox.min.js",
+  "keywords": [
+    "Lightbox",
+    "jQuery",
+    "swipebox"
+  ],
+  "author": "brutaldesign <hello@csag.co>",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brutaldesign/swipebox.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "+(css|js)/*swipebox*"
```

Comment:
We can directly put `"img/*"` under this line because their base path are the same.